src/lib/SX1280Driver/SX1280_hal.cpp

```diff
@@ -62,14 +62,12 @@ void SX1280Hal::init()
     tx1_enable_set_bits |= SET_BIT(GPIO_PIN_TX_ENABLE);
     tx1_enable_clr_bits |= SET_BIT(GPIO_PIN_RX_ENABLE);
     tx1_enable_clr_bits |= SET_BIT(GPIO_PIN_RX_ENABLE_2);
-    tx1_enable_clr_bits |= SET_BIT(GPIO_PIN_TX_ENABLE_2);
 
     tx2_enable_set_bits = 0;
     tx2_enable_clr_bits = 0;
     tx2_enable_set_bits |= SET_BIT(GPIO_PIN_PA_ENABLE);
     tx2_enable_set_bits |= SET_BIT(GPIO_PIN_TX_ENABLE_2);
     tx2_enable_clr_bits |= SET_BIT(GPIO_PIN_RX_ENABLE_2);
-    tx2_enable_clr_bits |= SET_BIT(GPIO_PIN_TX_ENABLE);
```

Comment:
@pkendall64 I know why you added these lines but they dont play well with Gemini mode :smiley:

Reply:
> @pkendall64 I know why you added these lines but they dont play well with Gemini mode 😃

Do what you need to do. Gemini is the new hot 💩

Reply:
Hmm, perhaps it would be nicer to have a `tx_all_enable_clr_bits` & `tx_all_enable_set_bits` instead, just so it's more readable.

Reply:
Done